resources/views/reports/custom.blade.php

```diff
@@ -289,6 +294,16 @@
               </div>
           </div>
 
+          <!-- Last Checkin Date -->
+          <div class="form-group checkin-range">
+              <label for="checkin_date" class="col-md-3 control-label">{{ trans('general.last') }} {{ trans('general.checkin') }} {{  trans('general.range') }}</label>
+              <div class="input-daterange input-group col-md-6" id="datepicker">
+                  <input type="text" class="form-control" name="checkin_date_start" aria-label="checkin_date_start">
+                  <span class="input-group-addon">to</span>
```

Comment:
Can we translate this, and/or just use a dash so we don't have to translate it?

Reply:
Translated. I noticed the other `to`s in the file aren't translated but I don't want to make this PR's diff too large. I'll open another PR to translate them once this is in.